pom.xml

```
@@ -14,8 +14,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>1.8</java.version>
-        <kotlin.version>1.3.72</kotlin.version>
-        <kotlin-coroutines.version>1.3.9</kotlin-coroutines.version>
+        <kotlin.version>1.4.10</kotlin.version>
```

There is a newer version since December 7, 2020 available. Why you use 1.4.10?
```suggestion
        <kotlin.version>1.4.21</kotlin.version>
```